js/parts/Axis.js

```diff
@@ -4627,9 +4627,15 @@ H.extend(Axis.prototype, /** @lends Highcharts.Axis.prototype */{
                 options.allowDecimals !== false
             );
 
-        // Find the tick positions. Work on a copy (#1565)
-        this.tickPositions = tickPositions =
-            tickPositionsOption && tickPositionsOption.slice();
+        /**
+         * Contains all ticks that are laid out on the axis.
+         *
```

Comment:
Note to self: Write a more concise explanation and warn that it should not be changed.